nautobot/dcim/tests/test_views.py

```diff
@@ -4670,9 +4672,15 @@ def test_correct_handling_for_model_protected_error(self):
             "confirm": True,
             "_confirm": True,  # Form button
         }
-        response = self.client.post(self._get_url("bulk_delete"), data, follow=True)
         # Assert protected error message included in the response body
-        self.assertBodyContains(response, f"<span>{device_type_to_software_image_file}</span>", html=True)
```

Comment:
Same comment/concern here. We may not need two of these test cases now that the logic is consolidated.